app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb

```diff
@@ -0,0 +1,68 @@
+<%#
+kind: PXEGrub
+name: Kickstart default PXEGrub
+oses:
+- CentOS
+- Fedora
+- RedHat
+-%>
+# This file was deployed via '<%= template_name %>' template
+
+<%
+  major = @host.operatingsystem.major.to_i
+  mac = @host.provision_interface.mac
+
+  # Tell Anaconda to perform network functions with boot interface
+  #  both current and legacy syntax provided
+  options = ["network", "ksdevice=bootif", "ks.device=bootif"]
+  if mac
+    bootif = '00-' + mac.gsub(':', '-')
+    options.push("BOOTIF=#{bootif}")
+  end
+
+  # Tell Anaconda what to pass off to kickstart server
+  #  both current and legacy syntax provided
+  options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
+
+  # handle non-DHCP environments (e.g. bootdisk)
+  subnet = @host.provision_interface.subnet
+  unless subnet.dhcp_boot_mode?
+    # static network configuration
+    ip = @host.provision_interface.ip
+    mask = subnet.mask
+    gw = subnet.gateway
+    dns = [subnet.dns_primary]
+    if subnet.dns_secondary != ''
+      dns.push(subnet.dns_secondary)
+    end
+    if (@host.operatingsystem.name.match(/Fedora/i) && major < 17) || major < 7
+      # old Anacoda found in Fedora 16 or RHEL 6 and older
```

Comment:
Must be fixed in community-templates -> https://github.com/theforeman/foreman/pull/4322